frontend/src/components/sign/components/redirect-after-sign/redirect-after-sign.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+import { StorageKey } from '@common/enums/app/storage-key.enum';
+import { Grid, CircularProgress } from '@mui/material';
+
+const RedirectAfterSign = (): React.ReactElement => {
+  const [searchParams] = useSearchParams();
+  const accessToken = searchParams.get('accessToken');
+  const refreshToken = searchParams.get('refreshToken');
+
+  useEffect(() => {
+    if (accessToken && refreshToken) {
+      localStorage.setItem(StorageKey.TOKEN, accessToken);
+      localStorage.setItem(StorageKey.REFRESH, refreshToken);
+    }
+    window.close();
+  }, []);
```

Comment:
You didn't pass the `accessToken` & `refreshToken` as the dependencies to the `useEffect`. Therefore, this `useEffect` will run _only_ on the first render, where the `at` and `rt` are `undefined`

Reply:
The question got resolved here - https://binary-academy-2022.slack.com/archives/D03TF60GL8H/p1661419055643939